src/anshuKJun20/StringPalindrome.java

```diff
@@ -0,0 +1,22 @@
+package anshuKJun20;
+
+public class StringPalindrome {
+
+	void finfPalindromeString(String name) {
+		String result = "";
+		for (int index = (name.length() - 1); index >= 0; index--) {
+			result = result + name.charAt(index);
+		}
+		if (name.equalsIgnoreCase(result)) {
```

Comment:
Use .equals() method only for palindrome. Because Radar is not Palindrome string.